### PR TITLE
docs: Update README.md to include layer Point of Contacts

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**
Now that there are multiple layers in this repo that are actively used, we need to make sure the people who understand each layer are documented and tagged on any Issues.

### CODE REVIEW

**Comment by KarenGhavam-lunarG** [comment]
@jeremyg-lunarg I believe we should get the github handle for the contributor of the timeline semaphore layer and add them to the contacts. And then create an issue assigned to that individual that they should add a document describing the purpose of the layer, similar to what exists for the sync2 and shader object layer.

**Review by KarenGhavam-lunarG — CHANGES_REQUESTED** [review]
@jeremyg-lunarg I believe we should get the github handle for the contributor of the timeline semaphore layer and add them to the contacts. And then create an issue assigned to that individual that they should add a document describing the purpose of the layer, similar to what exists for the sync2 and shader object layer.

**Comment by jeremyg-lunarg** [comment]
@llandwerlin-intel , are you OK being listed as a Point of Contact for VK_LAYER_khronos_timeline_semaphore?  It has had very little activity over the last couple of years but at least [one person](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/pull/118) is still using on Android.

**Comment by llandwerlin-intel** [comment]
> @llandwerlin-intel , are you OK being listed as a Point of Contact for VK_LAYER_khronos_timeline_semaphore? It has had very little activity over the last couple of years but at least [one person](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/pull/118) is still using on Android.

Not sure I would be able to do any work on it. But if there is no one else, sure.

**Comment by jeremyg-lunarg** [comment]
> > @llandwerlin-intel , are you OK being listed as a Point of Contact for VK_LAYER_khronos_timeline_semaphore? It has had very little activity over the last couple of years but at least [one person](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/pull/118) is still using on Android.
> 
> Not sure I would be able to do any work on it. But if there is no one else, sure.

Thank you. We ought to be able to handle build problems like the one I linked. We don't include this layer in the Vulkan SDK, so I really hope it continues to not need much attention.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,3 +42,15 @@ any new files need to have the Khronos license (Apache 2.0 style) and copyright
 Please see an existing file in this repository for an example.
 
 You can include your individual copyright after any existing copyrights.
+
+### **Point of Contact **
+
+If you are adding a new layer, you must provide a point of contact and list them in the README.md.
+The Point of Contact handle Issues reported against the layer, as the repository maintainers most
```
Comment (KarenGhavam-lunarG) [inline CONTRIBUTING.md:49]: The Point of Contact handles issues reported against the layer, ...

**Review by KarenGhavam-lunarG — COMMENTED** [review]
Only one small grammar change, but the rest looks good.

**Review by KarenGhavam-lunarG — APPROVED** [review]

**Pull request merged**